molecule/default/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: instance
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-centos7}-ansible:latest"
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    pre_build_image: true
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+scenario:
+  name: default
+  test_sequence:
+    - lint
+    - destroy
+    - dependency
+    - syntax
+    - create
+    - prepare
+    - converge
+    # - idempotence
```

Comment:
idempotence currently fails - I think for the deployment this is the `recreate: true`, not sure why the endpoint one fails. Bit hard to debug with molecule because it doesn't display the diffs - I'll fix this at some later date.
```
ERROR: Idempotence test failed because of the following tasks:
* [instance] => ansible-role-portainer : Deploy Portainer to instance
* [instance] => ansible-role-portainer : Define Endpoints
```